@here/olp-sdk-dataservice-api/lib/lookup-api.ts

```diff
@@ -180,5 +182,5 @@ export async function resourceAPIList(
         headers
     };
 
-    return builder.request<API[]>(urlBuilder, options);
+    return builder.requestBlob(urlBuilder, options);
```

Comment:
Please use deprecation instead of changes to avoid API breaks

Reply:
Done